vm/state_migration/src/common/macros/mod.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+mod system;
+mod verifier;
+
+#[macro_export(local_inner_macros)]
```

Comment:
Could you briefly explain why we need the `local_inner_macros`?

Reply:
I intended to use local macros inside local macros to further refactor some functions but it's not part of this PR, I can remove it from here and only add it later when needed